mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,20 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.removeOrder = (req, res) => { 
+  Order.findByIdAndDelete(req.params.orderId) 
+    .then(deletedOrder => { 
+      if (!deletedOrder) {
+        return res.status(404).json({
+          error: "Order not found", 
+        });
+      }
+      res.json({ message: "Order deleted successfully" }); 
+    })
+    .catch(error => { 
+      res.status(500).json({
```

Comment:
Inconsistent error handling. The error response format is not consistent with the rest of the codebase. In other parts of the code, the error response is in the format `{ error: errorHandler(err) }`, but here it's not specified.

Missing error message. The error response should include a meaningful error message to provide feedback to the client. 

Inconsistent status code. The status code 500 is used, but it's not clear why. In other parts of the code, status code 400 is used for error handling. There should be a consistent approach to status code usage throughout the codebase.